upgrade/upgrade.go

```diff
@@ -219,12 +224,14 @@ func doAPIVersionUpgrade(namespace string, config *restclient.Config, lhClient *
 	return nil
 }
 
-func upgradeLocalNode() (err error) {
+func upgradeLocalNode(lhVersionBeforeUpgrade string) (err error) {
```

Comment:
Do we still need this? We already check if upgrade path supported or not before. So from 1.5, the qualified version will be 1.4.x or 1.5.0. 

If yes, probably we can remove this old function.

Reply:
Removed this old function.